README.md

```diff
@@ -1,59 +1,162 @@
 
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=java-springboot-quickstart-repo&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/java-springboot-quickstart)
+Often, the first step developers take after creating their database is to create a REST API that can perform Create, Read, Update, and Delete (CRUD) operations for that database. This repo is designed to teach you and give you a starter project (in Java using Spring Boot) to generate such a REST API. After you have installed the travel-sample bucket in your database, you can run this application which is a REST API with Swagger documentation so that you can learn:
+
+1. How to create, read, update, and delete documents using Key-Value[ operations](https://docs.couchbase.com/java-sdk/current/howtos/kv-operations.html) (KV operations). KV operations are unique to Couchbase and provide super fast (think microseconds) queries.
+2. How to write simple parametrized [N1QL queries](https://docs.couchbase.com/java-sdk/current/howtos/n1ql-queries-with-sdk.html) using the built-in travel-sample bucket.
+
+Full documentation for the tutorial can be found on the [Couchbase Developer Portal](https://developer.couchbase.com/tutorial-quickstart-spring-boot-java/).
+
+If you are looking for a quick start using [Micronaut](https://micronaut.io/)](https://micronaut.io/), you can find it in this [repo](https://github.com/couchbase-examples/java-quickstart-micronaut).
 
-Full documentation can be found on the [Couchbase Developer Portal](https://developer.couchbase.com/tutorial-quickstart-java-springboot/).
 ## Prerequisites
+
 To run this prebuilt project, you will need:
 
-- Couchbase Server 7 Installed (version 7.0.0-5247 or higher)
-- Java SDK v1.8 or higher installed 
-- Code Editor installed (IntelliJ IDEA, Eclipse, or Visual Studio Code)
-- Maven command line
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/java-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+  - To run this tutorial using a self-managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- [Java](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) 11 or higher installed
+  - Ensure that the Java version is compatible with the Couchbase SDK.
+- Loading Travel Sample Bucket
+  If travel-sample is not loaded in your Capella cluster, you can load it by following the instructions for your Capella Cluster:
+  - [Load travel-sample bucket in Couchbase Capella](https://docs.couchbase.com/cloud/clusters/data-service/import-data-documents.html#import-sample-data)
 
-## Install Dependencies 
+## App Setup
 
+We will walk through the different steps required to get the application running.
 
-```sh
-mvn package
+### Cloning Repo
+
+```shell
+git clone https://github.com/couchbase-examples/java-springboot-quickstart
```

Comment:
The link is wrong

Reply:
the link is working. added .git at the end to match the command provided in github